### DISCUSSION TITLE
Creating language extension: invalid indents.scm

### DISCUSSION THREAD

**Discussion by artegoser** [zed-industries/zed#16197] [Feature Requests]
I'm creating an extension for my language, doing everything according to the documentation. But the file indents.scm is invalid. 

indents.scm: 
```scheme
(tags_body "]" @end) @indent
(el_tags "]]" @end) @indent

([
  (table_syntax)
  (paragraph)
  (mlmstext)
  (rmltext)
  (mltext)
] "}" @end) @indent
```

Logs: 

```txt
2024-08-14T09:09:45.008204031+03:00 [ERROR] failed to load language daleth:
Error loading indents query

Caused by:
    Query error at 1:13. Invalid node type 
```

After looking at other extensions I tried to copy indents.scm from them.

gleam/indents.scm:  
```scheme
(_ "[" "]" @end) @indent
(_ "{" "}" @end) @indent
(_ "(" ")" @end) @indent
```

Logs: 

```txt
2024-08-14T09:08:18.509549565+03:00 [ERROR] failed to load language daleth:
Error loading indents query

Caused by:
    Query error at 1:5. Invalid node type 
```

Without this file, the code highlighting works fine. Similar situation with brackets.scm file, but it is not so important, because they can be customized in config.toml.  

I don't understand what the problem is at all, if the same code works in other extensions, why doesn't it work in mine?

Probably because of this invalid file, indentation is not added in a new line (in helix it works fine):
![image](https://github.com/user-attachments/assets/fd7737d5-eaa5-4d05-bf45-1e8e2de5a914)

Extension: <https://github.com/TxtDot/daleth-zed>
Grammar: <https://github.com/TxtDot/tree-sitter-daleth>

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
